bindings/rust/src/lib.rs

```diff
@@ -691,7 +681,7 @@ mod tests {
         )
         */
         let input = hex::decode(
-        "0061736d010000000105016000017f030201000404017000000504010101020606017f0041000b07180403666f6f00000267310300037461620100036d656d02000a06010400412a0b").unwrap();
+            "0061736d010000000105016000017f030201000404017000000504010101020606017f0041000b07180403666f6f00000267310300037461620100036d656d02000a06010400412a0b").unwrap();
```

Comment:
Not sure what introduced this indent change?

Reply:
CLion.